addOns/automation/src/main/java/org/zaproxy/addon/automation/AbstractAutomationTest.java

```diff
@@ -41,7 +43,7 @@ public String toString() {
                 case WARN:
                     return Constant.messages.getString("automation.dialog.test.onfail.warn");
                 default:
-                    return null;
+                    return "";
```

Comment:
we can use the above constant only.

Reply:
You mean `EMPTY_SUMMARY`?

Reply:
I was thinking to rename the constant as EMPTY_STRING and use it at both places. It doesn't bring any benefit though and the string pool should handle the memory thing too. So I think can ignore this review comment.